modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/connection/ElasticsearchInstanceSettingsBuilder.java

```diff
@@ -305,7 +305,8 @@ private void _loadDefaultConfigurations() {
 
 		_configureTestMode();
 
-		put("transport.type", "netty4");
+		put("transport.type", "security4");
+		put("xpack.ml.enabled", false);
```

Comment:
In the final fix, we should also disable the rest of the x-pack features: https://issues.liferay.com/browse/LPS-145631?focusedCommentId=2606735&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2606735

Reply:
Also, do we really need to hard-code anything? Why don't we just ship the `elasticsearch.yml` with the connector and move it to the ES_HOME/config after sidecar has been extracted. :thinking: